ck3-tiger/Cargo.toml

```diff
@@ -15,17 +15,15 @@ rust-version = "1.70"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-tiger-lib = { version = "0.9.5", path = "..", features = ["ck3"] }
+tiger-lib = { path = "..", features = ["ck3"] }
```

Comment:
Hmm you removed the version. Will this still pull in the latest tiger-lib when updating ck3-tiger?

Reply:
yes you are right won't be able to publish on crates.io otherwise. will fix